src/renderer/services/labkey-client/index.ts

```diff
@@ -177,16 +179,12 @@ export default class LabkeyClient extends HttpCacheClient {
     column: string,
     searchString = ""
   ): Promise<string[]> {
-    const additionalQueries = [
-      `query.columns=${column}`,
-      `query.sort=${column}`,
-      `query.maxRows=100`,
-    ];
+    const additionalQueries = [`query.sort=${column}`, `query.maxRows=100`];
     if (!isEmpty(searchString)) {
       // The fms.file table is so large that running "contains" queries on it will take forever, so use "eq" instead
       /*
        TODO:
-        In a more perfect world, annotations that reference other files (specifically, files fitting certain annotation
+        In a more perfect world, annotations that reference other files (specifically files fitting certain annotation
```

Comment:
nit i think it reads better with the ','

Reply:
Obviously it doesn't matter very much, but I think I removed the comma here because I don't like repeating sentence structures one after the other, and I thought the stuff in parentheses read that way.
e.g.
`Beginning 1, main point 1. Beginning 2, main point 2`